libsrc/Makefile

```diff
@@ -14,14 +14,16 @@ CBMS = c128   \
 GEOS = geos-apple \
        geos-cbm
 
-TARGETS = apple2    \
+TARGETS = \
+          $(CBMS)   \
+          $(GEOS)   \
+          apple2    \
           apple2enh \
           atari     \
-          atarixl   \
           atari5200 \
+          atarixl   \
```

Comment:
This pull request isn't related to Atari stuff so please leave Atari stuff alone. I strongly disgree to mix "general cleanup" operations with significant changes. Apart from that the order of Atari items was carefully chosen.